spec/httpserver/utilities/arguments_spec.clj

```diff
@@ -0,0 +1,24 @@
+(ns httpserver.arguments-spec
+ (:require [speclj.core :refer :all]
+           [httpserver.utilities.arguments :as argumenetParser]
+           [clojure.tools.cli :refer [parse-opts]]))
+
+(def args '("-p" "5000" "-d" "./site"))
+(def argsEmpty '())
```

Comment:
I'd like to see you get in the habit of using `let` more often than `def`ing things. You can have nested `let`s and can bind variables with `let` over any number of forms. For example:

```clojure
(describe "foo"
  (let [args ["-p" "5000" "-d" "./site"]]
    (it "sets port to 5000"
      ...)
    (it "sets directory to ./site"
      (let [something (something-else)]
        ...)))
  (let [empty-args '()]
    ...))
```